### PR TITLE
DomainServices: Add missing properties, add examples, and fix linting errors

### PR DESCRIPTION
**Pull request by jihochang**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/domainservices/resource-manager/Microsoft.AAD/stable/2017-01-01/domainservices.json [hunk 1]
```diff
@@ -51,7 +51,8 @@
           "Get Operations": {
             "$ref": "./examples/GetOperations.json"
           }
-        }
+        },
+        "x-ms-pageable": {}
```
Comment (sarangan12) [inline specification/domainservices/resource-manager/Microsoft.AAD/stable/2017-01-01/domainservices.json]: What is the nextLinkName?

File: specification/domainservices/resource-manager/Microsoft.AAD/stable/2017-01-01/domainservices.json [hunk 2]
```diff
@@ -343,7 +344,8 @@
         },
         "location": {
           "type": "string",
-          "description": "Resource location"
+          "description": "Resource location",
+		  "x-ms-mutability":["read", "create"]
```
Comment (sarangan12) [inline specification/domainservices/resource-manager/Microsoft.AAD/stable/2017-01-01/domainservices.json]: nit: spacing

**Review by sarangan12 — CHANGES_REQUESTED** [review]

File: specification/domainservices/resource-manager/Microsoft.AAD/stable/2017-01-01/domainservices.json (hunk 2 above)
Comment (jihochang) [reply specification/domainservices/resource-manager/Microsoft.AAD/stable/2017-01-01/domainservices.json]: Fixing

**Comment by jihochang** [comment]
Could you take another look? I fixed your comments!

File: specification/domainservices/resource-manager/Microsoft.AAD/stable/2017-01-01/domainservices.json (hunk 1 above)
Comment (jihochang) [reply specification/domainservices/resource-manager/Microsoft.AAD/stable/2017-01-01/domainservices.json]: Updated

**Comment by jihochang** [comment]
Ping for @sarangan12 to take second look.

**Review by sarangan12 — APPROVED** [review]

**Pull request merged**